generators/chipyard/src/main/scala/config/RocketConfigs.scala

```diff
@@ -115,3 +115,13 @@ class SV48RocketConfig extends Config(
   new freechips.rocketchip.rocket.WithSV48 ++
   new freechips.rocketchip.rocket.WithNHugeCores(1) ++
   new chipyard.config.AbstractConfig)
+
+class LTraceEncoderRocketConfig extends Config(
```

Comment:
Can you add a brief comment to this class (similar to previous ones) indicating what it does?